lib/package.gi

```diff
@@ -865,7 +865,7 @@ InstallGlobalFunction( PackageAvailabilityInfo,
 
       # Test whether the availability test function fits.
       GAPInfo.PackageCurrent:= inforec;
-      test:= inforec.AvailabilityTest();
+      test:= (not IsBound(inforec.AvailabilityTest)) or inforec.AvailabilityTest();
```

Comment:
```suggestion
      test:= (not IsBound(inforec.AvailabilityTest)) or (inforec.AvailabilityTest() = true);
```
Apparently there are packages for which `AvailabilityTest()` is something different from `true` or `false`, and this was accepted up to now; but the `or` construction needs `true` or `false`.

Reply:
Indeed it seems almost 20 packages return `fail` instead of `false`. Ah well.